app/src/main/java/edu/usf/sas/pal/muser/firebase/models/SongActionProvider.java

```diff
@@ -0,0 +1,16 @@
+package edu.usf.sas.pal.muser.firebase.models;
+
+/**
+ *  The provider class that stores different types of action that can be performed by the user
+ *  on a sound track.
+ */
+public interface SongActionProvider {
+    @interface ActionType{
+        String ACTION_PLAY = "Play";
```

Comment:
Rename `ACTION_*` to `EVENT_*`.

Reply:
I think I changed them in the same commit https://github.com/CUTR-at-USF/MUSER/pull/41/commits/aa979a6b0caaa05366a642e0a1c64d8a1ad544a9 which replaces `ACTION` with `EVENT`.